x-pack/plugins/ml/public/application/jobs/jobs_list/components/edit_job_flyout/edit_job_flyout.js

```diff
@@ -48,6 +48,8 @@ export class EditJobFlyoutUI extends Component {
       jobDescription: '',
       jobGroups: [],
       jobModelMemoryLimit: '',
+      jobModelSnapshotRetentionDays: 1,
```

Comment:
Is this the default for a brand new job?  If so, it should be 10 for 7.8 and above.  (The default for a brand new job was 1 before 7.8.)

Reply:
changed in b2e0bcb76cd21cf77c8fe51e71b058b689c456a1